cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/ldap/LdapConfig.java

```diff
@@ -90,7 +91,7 @@
 {
     private static final Logger logger = Logger.getLogger(LdapConfig.class);
 
-    public static final String CONFIG = LdapConfig.class.getSimpleName() + ".properties";
+    public static final String CONFIG = "ldap-config.properties";
```

Comment:
Although the naming style is improved, for other services we have implemented a scheme where the service name (ac- in our case) is prepended to the config file name so that ops people can tell which config file(s) go with each deployed service. That change is a little more invasive because you need to propagate the service name (or entire config file name) from a web resource that can access the request (in the case of cadc-access-control-server it is extra painful because there are a bunch of bare servlet impls and no base action class).... maybe create an issue in github for this because it is pretty painful and refactoring the library to use cadc-rest would make it much easier.

For now, change the file name to ac-ldap-config.properties and put a comment beside it that it is a temporary hack.